ios/RNCallKeep/RNCallKeep.m

```diff
@@ -119,18 +118,25 @@ - (void)dealloc
     ];
 }
 
+- (bool)hasListeners
+{
+    NSInteger _listenerCount = [[self valueForKey:@"_listenerCount"] intValue];
+
+    if (_listenerCount == 0) return NO;
+    else return YES;
+}
+
 - (void)startObserving
 {
     NSLog(@"[RNCallKeep][startObserving]");
-    _hasListeners = YES;
     if ([_delayedEvents count] > 0) {
         [self sendEventWithName:RNCallKeepDidLoadWithEvents body:_delayedEvents];
     }
 }
 
 - (void)stopObserving
 {
-    _hasListeners = FALSE;
+    // Silence is golden.
```

Comment:
These changes would have same effect as commented out line with `_hasListeners = FALSE;`.
If you go with this I would suggest still to have `[self setValue:@0 forKey:@"_listenerCount"];` here also.

Reply:
@Romick2005 yeah I agree. After some thoughts I think we should just go for #574 since its much more simpler to maintain and exposes RNCallKeep to more "gracefully" break if RTCEventEmitter changes its internals.